packages/manager/modules/pci/src/components/project/flavors-list/translations/Messages_en_GB.json

```diff
@@ -1,23 +1,19 @@
 {
   "pci_project_flavors_category_definition_balanced": "Instances for general use offer a good balance of RAM and performance.",
+  "pci_project_flavors_category_definition_baremetal": "Bare metal instances offer hardware servers on demand. They are delivered in a few minutes, and can be billed on an hourly or monthly basis.",
   "pci_project_flavors_category_definition_cpu": "Instances with optimised computing are ideally suited to applications that require high processing speeds or parallel task processing.",
   "pci_project_flavors_category_definition_ram": "Instances with optimised memory are recommended for your databases, analyses and in-memory computing, as well as other RAM-intensive applications.",
   "pci_project_flavors_category_definition_accelerated": "Instances with accelerated computing (GPU, FPGA) are up to 1,000 times faster than a CPU for certain applications (rendering, video transcoding, bioinformatics, big data, deep learning, etc.)",
   "pci_project_flavors_category_definition_vps": "Instances with shared resources (Sandbox) are suitable for testing, verifications and development environments. Their performance may vary slightly over time.",
+  "pci_project_flavors_category_definition_iops": "IOPS instances deliver the fastest disk transactions in the Public Cloud range.",
   "pci_project_flavors_category_definition_none": "No model available for this category",
   "pci_project_flavors_disk_size": "The disk space is lower than that of the current model",
   "pci_project_flavors_spec_ram": "{{ram}} RAM",
-  "pci_project_flavors_spec_core": "Processor:",
```

Comment:
Removed in https://github.com/ovh/manager/commit/3caac03dd48ed032fac0a04c59ead200784f0443#diff-2cc2fa838250716e9b4ede7fe8281127